### ISSUE TITLE
Collect and View Browser Plugin Information

### ISSUE THREAD

**Issue by justaskingquestions** [pentestgeek/phishing-frenzy#389]
How can I collect and report informaiton about the Browser Plugins (Adobe Flash, Shockwave, Java etc.) of users visiting the Links in my phishing campaign?

**Comment by zeknox** [comment]
Likely the best way to get this info is to download the raw apache logs from the Campaign > Reports section.  Within the raw Apache logs would have all the Browser details including plugins. You'd likely need to parse out the data you're looking for.

**Comment by justaskingquestions** [comment]
Thanks for you quick reply. It is not the user-agent string that I am after but things like the versions of certain plugins. Information that could be used in preparation for a red-team exercise where the users are targeted with browser-/plugin-specific exploits. As far as I am aware such info would probably have to be collected using JS such as the code used at [PinLady](http://www.pinlady.net/PluginDetect/All/)

**Comment by zeknox** [comment]
Yup you're correct, we do not have any collection methods via JS but certainly could be incorporated.
